docs/source/blogs/variant-normalization-blog/variant-normalization-blog.rst

```diff
@@ -54,7 +54,7 @@ The ``normalize_variants`` transformer brings unprecedented scalability and simp
 The underlying normalization algorithm and its accuracy
 -------------------------------------------------------
 
-There are several single-node tools for variant normalization that use different normalization algorithms. Widely used tools for variant normalization include `vt normalize <https://genome.sph.umich.edu/wiki/Vt>`_, `bcftools norm <https://www.htslib.org/doc/bcftools.1.html#norm>`_, and the GATK’s `LeftAlignAndTrimVariants <https://software.broadinstitute.org/gatk/documentation/tooldocs/3.8-0/org_broadinstitute_gatk_tools_walkers_variantutils_LeftAlignAndTrimVariants.php>`_.
+There are several single-node tools for variant normalization that use different normalization algorithms. Widely used tools for variant normalization include `vt normalize <https://genome.sph.umich.edu/wiki/Vt>`_, `bcftools norm <https://www.htslib.org/doc/bcftools.html#norm>`_, and the GATK’s `LeftAlignAndTrimVariants <https://gatk.broadinstitute.org/hc/en-us/articles/360036857271-LeftAlignAndTrimVariants>`_.
```

Comment:
Let's refer to the latest version while we are at it. https://gatk.broadinstitute.org/hc/en-us/articles/360037225872-LeftAlignAndTrimVariants